ex5B/ex5Spec.js

```diff
@@ -0,0 +1,30 @@
+var firstUniqueChar = require('./ex5.js');
+
+describe("function firstUniqueChar", function() {
+	it("finds another unique character e in the string", function() {
+		expect(firstUniqueChar("abcdedcba")).toBe("e");
+	});
+
+	it("return null when we only have one [non-unique] char", function() {
+		expect(firstUniqueChar("abcdeedcba")).toBeNull();
+	});
+
+	it("finds another unique character e in the string", function() {
+		expect(firstUniqueChar("abababcab")).toBe("c");
+	});
+
+	it("return null, all chars are same", function() {
+		expect(firstUniqueChar("aaaaaaaaa")).toBeNull();
+	});
+	it("works on really long strings", function() {
+		var longString = '';
+		for (i=100;i<100000;i++) {
+			longString += String.fromCharCode(i);
+		}
+		firstUniqueChar(longString);
+	});
+
+	it("works for a single character", function() {
+		expect(firstUniqueChar("a")).toBe("a");
+	});
+});
```

Comment:
Please add a few tests when you have more than one non-repeating characters, and you will see they don't work as expected (they don't always return the *first* one).